sdk/rust-examples/huffman-encoding/src/main.rs

```diff
@@ -0,0 +1,178 @@
+//! Huffman Encoding Example
```

Comment:
Run `cargo fmt` for this example, before the merging. I think some of the formats is C-style rather than Rust-style. The `cargo fmt` will automatically format for you.

Reply:
Yes, I will do this before committing the changes again!👍👍

Reply:
I've fixed this issue now. Thank you for the suggestion!!